apps/api/e2e/tests/api-key-retrieval.test.ts

```diff
@@ -42,7 +42,7 @@ describe("API Key Retrieval", () => {
     await adminClient.loginAsAdmin(adminApiKey);
 
     // Register a new team
-    const { team, apiKey } = await registerTeamAndGetClient(adminClient);
+    const { team, apiKey } = await registerTeamAndGetClient(adminApiKey);
```

Comment:
I updated this helper to use the SDK.  Because of this you will see changes to all of the test files, but all of these changes are just updating the argument and some return types